### PR TITLE
minor changes: multiple origins, email auth followups

### PR DESCRIPTION
**Pull request by andrewkmin**
$title (followups for https://github.com/tkhq/demo-passkey-wallet/pull/13)

allowing multiple origins, as it turns out, is useful for situations where we'd want to test changes to the frontend that aren't pushed to main.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -320,11 +322,13 @@ func main() {
 		txHash, err := ethereum.BroadcastTransaction(signedDropTx)
 		if err != nil {
 			ctx.String(http.StatusInternalServerError, errors.Wrap(err, "unable to broadcast drop transfer").Error())
+			return
```
Comment (andrewkmin) [inline main.go:325]: to prevent recording drops when broadcasting fails

File: frontend/utils/urls.ts [hunk 2]
```diff
@@ -48,10 +48,6 @@ export function sendTxUrl(): string {
   return BACKEND_API_BASE_URL + "/api/wallet/send-tx";
 }
 
-export function broadcastTxUrl(): string {
```
Comment (andrewkmin) [inline frontend/utils/urls.ts:51]: removing this endpoint; for the sake of example, all sign-related requests should be stamped and forwarded to a single endpoint `/api/wallet/send-tx`

File: main.go [hunk 3]
```diff
@@ -78,7 +80,7 @@ func main() {
 	router.Use(ginErrorLogMiddleware)
 
 	router.Use(cors.New(cors.Config{
-		AllowOrigins:     []string{clientOrigin},
+		AllowOrigins:     origins,
```
Comment (r-n-o) [inline main.go:83]: beautiful

**Review by r-n-o — APPROVED** [review]
💯

**Pull request merged**